packages/@headlessui-react/playground-utils/hooks/use-popper.ts

```diff
@@ -1,18 +1,18 @@
-import React from 'react'
+import { useCallback, useRef, useMemo, RefCallback } from 'react'
```

Comment:
This file is not in Next.js pages.
We may not be able to change it until tsdx support new transform.